### DISCUSSION TITLE
alert service

### DISCUSSION THREAD

**Discussion by ghost** [weather-gov/api#336] [Q&A]
Hello,
I have been accessing NWS alerts as plain text reports using addresses like this:
    https://alerts-v2.weather.gov/products/NWS-IDP-PROD-4276808-3600205
For a while now it is giving a response like "Cannot GET /products/NWS-IDP-PROD-4276808-3600205".
Has this method of retrieving alerts retired or is it just temporarily unavailable?
I notice this api method works:
    https://api.weather.gov/alerts/NWS-IDP-PROD-4276808-3600205
But to use this I would have to rewrite code for the difference in formats (plain text vs json).
Thank you for providing such valuable services!

**Comment by scadergit** [comment]
As of today, alerts.weather.gov is still the operational version of Alerts. Recently, the preview of the next version Alerts (v2) was greatly revised, including a new url pattern. The site is now a single page application, but you can still build the url manually. Your example product would look like this:

https://alerts-v2.weather.gov/#/?id=NWS-IDP-PROD-4276808-3600205

I'm not certain what you consider to be a plain text report given neither version of the preview of v2 supported that format. The v2 versions are considered "presentation layers" and rendered using HTML for viewing with a web browser. Using the site to scrape product text instead of using the API is discouraged. With v2, the NWS has fully shifted to [the CAP XML specification](https://www.weather.gov/documentation/services-web-alerts). The API provides this format along with the default JSON.

**Comment by ghost** [comment]
Thanks Jared for your prompt reply!
Yes, by "plain text" I was meaning the presentation layer you provide, which I just link to if people want to see the full alert details.

________________________________
